src/style/content/conversation/input-bar.less

```diff
@@ -164,7 +164,7 @@
     color: var(--white);
 
     > svg {
-      margin-left: 0.3em;
+      margin-left: 4px;
```

Comment:
looks counter intuitive to be introducing px values in this PR but this shouldn't be responsive with font-size changes